src/interfaces/businessContact-interface.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * See:
+ * https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/contact_point.json
+ */
+export interface ContactPointIF{
```

Comment:
This is also named according to the schema (different in Edit UI).

Reply:
We should probably remove `ContactIF` and refer to this Schema as well, where used.

Reply:
I think just in the Contact component.

Reply:
It's used in the Alteration IF.

Reply:
ah ok. 
I believe it is the same IF, so we should delete the ContactIF and refer to this one perhaps.

Reply:
Yes. That's what drove me to do this -- I didn't want to add a bunch of duplicate IFs in Filings UI. So now I will just import the common/shared ones.